[2025-04-25 20:51:49,527: INFO: test_run: lets go]
[2025-04-25 20:53:08,769: INFO: test_run: lets get the party going]
[2025-04-25 21:00:31,938: INFO: test_run: lets get the party going]
[2025-04-26 11:32:41,549: INFO: test_run: lets get the party going]
[2025-04-26 16:30:45,759: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:30:45,767: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:32:54,444: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:32:54,448: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:33:26,752: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:33:26,755: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:33:38,039: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:33:38,043: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:40:33,135: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:40:33,140: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:40:38,196: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:40:38,201: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:41:25,466: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:41:25,469: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:43:46,191: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:43:46,195: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:45:10,646: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:45:10,649: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:46:30,077: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:46:30,079: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:46:30,080: INFO: common_func: created directory at: artifacts]
[2025-04-26 16:47:20,860: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:47:20,862: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:47:20,863: INFO: common_func: created directory at: artifacts]
[2025-04-26 16:48:50,574: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:48:50,576: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:48:50,577: INFO: common_func: created directory at: artifacts]
[2025-04-26 16:49:38,234: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:49:38,236: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:49:38,236: INFO: common_func: created directory at: artifacts]
[2025-04-26 16:51:47,247: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:51:47,251: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:51:47,252: INFO: common_func: created directory at: artifacts]
[2025-04-26 16:55:54,150: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:55:54,154: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:55:54,155: INFO: common_func: created directory at: artifacts]
[2025-04-26 16:55:54,156: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 16:58:01,163: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:58:01,167: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:58:01,169: INFO: common_func: created directory at: artifacts]
[2025-04-26 16:58:01,169: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 16:58:42,699: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 16:58:42,705: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 16:58:42,706: INFO: common_func: created directory at: artifacts]
[2025-04-26 16:58:42,707: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 17:00:27,073: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 17:00:27,077: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 17:00:27,080: INFO: common_func: created directory at: artifacts]
[2025-04-26 17:00:27,080: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 17:00:51,225: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 17:00:51,229: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 17:00:51,267: INFO: common_func: created directory at: artifacts]
[2025-04-26 17:00:51,269: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 17:00:54,014: INFO: 154209813: succesfully downloaded the zip artifacts/data_ingestion/data.zip with following info 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 85E5:5EF63:16A4B7B:1948A3E:680CF523
Accept-Ranges: bytes
Date: Sat, 26 Apr 2025 15:00:52 GMT
Via: 1.1 varnish
X-Served-By: cache-fra-etou8220174-FRA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1745679652.858839,VS0,VE397
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 809b3e13e20446288a7fb7ae1ccdfe8ad38e1f90
Expires: Sat, 26 Apr 2025 15:05:52 GMT
Source-Age: 0

]
[2025-04-26 23:04:48,532: INFO: test_run: Trying to copy data set: This is the Fzrst stage: data ingestion stay tuned and relax]
[2025-04-26 23:04:48,534: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 23:04:48,536: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 23:04:48,536: INFO: common_func: created directory at: artifacts]
[2025-04-26 23:04:48,536: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 23:05:02,918: INFO: test_run: Trying to copy data set: This is the Fzrst stage: data ingestion stay tuned and relax]
[2025-04-26 23:05:02,920: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 23:05:02,922: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 23:05:02,923: INFO: common_func: created directory at: artifacts]
[2025-04-26 23:05:02,923: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 23:05:02,923: ERROR: test_run: local variable 'filename' referenced before assignment]
Traceback (most recent call last):
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/test_run.py", line 9, in <module>
    pipeline.main()
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/src/clasifier_cnn/pipeline/stage01_data_ingestion.py", line 13, in main
    data_ingestion.download_zip_file()
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/src/clasifier_cnn/components/data_ingestion.py", line 21, in download_zip_file
    logger.info(f"The file {filename} already exists and the size is {get_size(Path(self.config.local_data_file))}")
UnboundLocalError: local variable 'filename' referenced before assignment
[2025-04-26 23:06:10,098: INFO: test_run: Trying to copy data set: This is the Fzrst stage: data ingestion stay tuned and relax]
[2025-04-26 23:06:10,099: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 23:06:10,101: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 23:06:10,102: INFO: common_func: created directory at: artifacts]
[2025-04-26 23:06:10,102: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 23:06:10,102: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-04-26 23:06:10,284: INFO: test_run: succefuly unziped the file and you are done with Fzrst stage: data ingestion]
[2025-04-26 23:06:45,900: INFO: test_run: Trying to copy data set: This is the Fzrst stage: data ingestion stay tuned and relax]
[2025-04-26 23:06:45,903: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 23:06:45,905: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 23:06:45,905: INFO: common_func: created directory at: artifacts]
[2025-04-26 23:06:45,905: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 23:06:48,572: INFO: data_ingestion: succesfully downloaded the zip artifacts/data_ingestion/data.zip with following info 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: E558:56E7B:1D766BA:20A77FD:680D4AE6
Accept-Ranges: bytes
Date: Sat, 26 Apr 2025 21:06:46 GMT
Via: 1.1 varnish
X-Served-By: cache-fra-eddf8230050-FRA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1745701606.402214,VS0,VE459
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 71d9dc4dfc3bc72526601b0cb5b4a13495822a22
Expires: Sat, 26 Apr 2025 21:11:46 GMT
Source-Age: 0

]
[2025-04-26 23:06:48,716: INFO: test_run: succefuly unziped the file and you are done with Fzrst stage: data ingestion]
[2025-04-26 23:08:51,395: INFO: test_run: Trying to copy data set: This is the Fzrst stage: data ingestion stay tuned and relax]
[2025-04-26 23:08:51,398: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-26 23:08:51,401: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-26 23:08:51,401: INFO: common_func: created directory at: artifacts]
[2025-04-26 23:08:51,402: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-04-26 23:08:53,454: INFO: data_ingestion: succesfully downloaded the zip artifacts/data_ingestion/data.zip with following info 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: E558:56E7B:1D766BA:20A77FD:680D4AE6
Accept-Ranges: bytes
Date: Sat, 26 Apr 2025 21:08:51 GMT
Via: 1.1 varnish
X-Served-By: cache-fra-eddf8230148-FRA
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1745701732.787013,VS0,VE1
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 6ad43e014f00e2219f8ae925a75f61e8972383a1
Expires: Sat, 26 Apr 2025 21:13:51 GMT
Source-Age: 125

]
[2025-04-26 23:08:53,601: INFO: test_run: succefuly unziped the file and you are done with Fzrst stage: data ingestion]
[2025-04-28 22:51:41,244: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-28 22:51:41,269: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-28 22:51:41,277: INFO: common_func: created directory at: artifacts]
[2025-04-28 22:52:56,579: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-04-28 22:52:56,584: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-04-28 22:52:56,589: INFO: common_func: created directory at: artifacts]
[2025-04-28 22:52:56,591: INFO: common_func: created directory at: artifacts/prepare_base_model]
[2025-04-28 22:53:20,506: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-04-28 22:53:20,726: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-04 17:13:53,437: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-04 17:13:53,439: ERROR: test_run: DataIngwationPipeline.main() takes 0 positional arguments but 1 was given]
Traceback (most recent call last):
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/test_run.py", line 10, in <module>
    pipeline.main()
TypeError: DataIngwationPipeline.main() takes 0 positional arguments but 1 was given
[2025-05-04 17:14:51,748: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-04 17:14:51,750: ERROR: test_run: DataIngwationPipeline.main() takes 0 positional arguments but 1 was given]
Traceback (most recent call last):
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/test_run.py", line 10, in <module>
    pipeline.main()
TypeError: DataIngwationPipeline.main() takes 0 positional arguments but 1 was given
[2025-05-04 17:15:30,659: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-04 17:15:30,661: ERROR: test_run: DataIngwationPipeline.main() takes 0 positional arguments but 1 was given]
Traceback (most recent call last):
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/test_run.py", line 10, in <module>
    pipeline.main()
TypeError: DataIngwationPipeline.main() takes 0 positional arguments but 1 was given
[2025-05-04 17:18:59,848: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-04 17:18:59,851: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-04 17:18:59,853: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-04 17:18:59,854: INFO: common_func: created directory at: artifacts]
[2025-05-04 17:18:59,854: ERROR: test_run: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/test_run.py", line 10, in <module>
    pipeline.main()
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/src/clasifier_cnn/pipeline/stage01_data_ingestion.py", line 11, in main
    data_ingestion_config = config.get_data_ingestion_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'
[2025-05-04 17:20:43,206: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-04 17:20:43,208: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-04 17:20:43,210: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-04 17:20:43,211: INFO: common_func: created directory at: artifacts]
[2025-05-04 17:20:43,211: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-04 17:20:46,462: INFO: data_ingestion: succesfully downloaded the zip artifacts/data_ingestion/data.zip with following info 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 9916:2E0CC7:2D5397E:3488AA4:681785CB
Accept-Ranges: bytes
Date: Sun, 04 May 2025 15:20:44 GMT
Via: 1.1 varnish
X-Served-By: cache-fra-eddf8230030-FRA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1746372044.003777,VS0,VE522
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 79322c6ec1e50a6d040cc2f7293a2a42b8f0b231
Expires: Sun, 04 May 2025 15:25:44 GMT
Source-Age: 0

]
[2025-05-04 17:20:46,567: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-04 17:22:40,678: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-04 17:22:40,679: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-04 17:22:40,681: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-04 17:22:40,682: INFO: common_func: created directory at: artifacts]
[2025-05-04 17:22:40,682: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-04 17:22:40,682: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-04 17:22:40,813: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-04 17:22:40,813: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-04 17:22:40,815: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-04 17:22:40,815: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-04 17:22:40,815: INFO: common_func: created directory at: artifacts]
[2025-05-04 17:22:40,815: INFO: common_func: created directory at: artifacts/prepare_base_model]
[2025-05-04 17:22:41,126: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-04 17:22:41,281: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-04 17:22:41,452: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-05 19:53:13,547: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-05 19:53:13,559: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-05 19:53:13,562: INFO: common_func: created directory at: artifacts]
[2025-05-05 19:56:14,829: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-05 19:56:14,835: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-05 19:56:14,838: INFO: common_func: created directory at: artifacts]
[2025-05-05 19:57:31,866: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-05 19:57:31,873: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-05 19:57:31,876: INFO: common_func: created directory at: artifacts]
[2025-05-05 19:58:34,372: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-05 19:58:34,392: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-05 19:58:34,470: INFO: common_func: created directory at: artifacts]
[2025-05-05 19:59:43,735: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-05 19:59:43,739: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-05 19:59:43,740: INFO: common_func: created directory at: artifacts]
[2025-05-05 20:00:32,238: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-05 20:00:32,244: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-05 20:00:32,249: INFO: common_func: created directory at: artifacts]
[2025-05-05 20:00:32,253: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-05 20:00:32,254: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-05 20:00:38,694: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-05 20:00:38,697: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-05 20:00:38,698: INFO: common_func: created directory at: artifacts]
[2025-05-05 20:00:38,698: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-05 20:00:38,699: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-05 20:00:50,085: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-05 20:00:50,087: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-05 20:00:50,088: INFO: common_func: created directory at: artifacts]
[2025-05-05 20:00:50,089: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-05 20:00:50,090: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-05 20:01:04,327: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-05 20:01:04,331: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-05 20:01:04,332: INFO: common_func: created directory at: artifacts]
[2025-05-05 20:01:04,333: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-05 20:01:04,334: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-09 17:55:20,313: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-09 17:55:20,315: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 17:55:20,318: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 17:55:20,319: INFO: common_func: created directory at: artifacts]
[2025-05-09 17:55:20,319: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-09 17:55:20,319: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-09 17:55:20,435: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-09 17:55:20,435: INFO: test_run: Trying to copy data set: This is the Second stage: Model Creation stay tuned and relax]
[2025-05-09 17:55:20,436: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 17:55:20,436: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 17:55:20,437: INFO: common_func: created directory at: artifacts]
[2025-05-09 17:55:20,437: INFO: common_func: created directory at: artifacts/prepare_base_model]
[2025-05-09 17:55:20,738: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 17:55:20,878: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 17:55:21,065: INFO: test_run: succefuly unziped the file and you are done with Second stage: Model Creation]
[2025-05-09 17:55:21,066: INFO: test_run: Trying to copy data set: This is the Third stage: Model Training stay tuned and relax]
[2025-05-09 17:55:21,069: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 17:55:21,070: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 17:55:21,070: INFO: common_func: created directory at: artifacts]
[2025-05-09 17:55:21,071: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-09 17:55:21,071: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-09 17:55:21,072: INFO: common_func: created directory at: artifacts/training]
[2025-05-09 17:55:21,214: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-09 17:55:21,737: ERROR: test_run: numpy() is only available when eager execution is enabled.]
Traceback (most recent call last):
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/test_run.py", line 38, in <module>
    pipeline.main()
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/src/clasifier_cnn/pipeline/stage_03_training.py", line 22, in main
    training.train(
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/src/clasifier_cnn/components/training.py", line 69, in train
    self.model.fit(
  File "/Users/rohithkamath/Documents/endtoendproject/chicken3/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py", line 122, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/Users/rohithkamath/Documents/endtoendproject/chicken3/lib/python3.10/site-packages/keras/src/backend/tensorflow/core.py", line 158, in convert_to_numpy
    return np.array(x)
NotImplementedError: numpy() is only available when eager execution is enabled.
[2025-05-09 17:57:47,496: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-09 17:57:47,498: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 17:57:47,499: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 17:57:47,499: INFO: common_func: created directory at: artifacts]
[2025-05-09 17:57:47,499: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-09 17:57:47,499: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-09 17:57:47,594: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-09 17:57:47,594: INFO: test_run: Trying to copy data set: This is the Second stage: Model Creation stay tuned and relax]
[2025-05-09 17:57:47,595: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 17:57:47,596: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 17:57:47,596: INFO: common_func: created directory at: artifacts]
[2025-05-09 17:57:47,596: INFO: common_func: created directory at: artifacts/prepare_base_model]
[2025-05-09 17:57:47,855: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 17:57:47,947: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 17:57:48,022: INFO: test_run: succefuly unziped the file and you are done with Second stage: Model Creation]
[2025-05-09 17:57:48,022: INFO: test_run: Trying to copy data set: This is the Third stage: Model Training stay tuned and relax]
[2025-05-09 17:57:48,024: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 17:57:48,026: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 17:57:48,026: INFO: common_func: created directory at: artifacts]
[2025-05-09 17:57:48,026: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-09 17:57:48,027: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-09 17:57:48,027: INFO: common_func: created directory at: artifacts/training]
[2025-05-09 17:57:48,241: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-09 17:57:55,139: ERROR: test_run: Unknown variable: <Variable path=dense/kernel, shape=(25088, 2), dtype=float32, value=[[ 0.011897   -0.00889055]
 [-0.00879171  0.01049202]
 [ 0.01110586  0.00455226]
 ...
 [-0.00044798  0.01047834]
 [-0.00620989  0.0112009 ]
 [ 0.00195403  0.00393406]]>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.]
Traceback (most recent call last):
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/test_run.py", line 38, in <module>
    pipeline.main()
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/src/clasifier_cnn/pipeline/stage_03_training.py", line 22, in main
    training.train(
  File "/Users/rohithkamath/Documents/endtoendproject/Chicken-Disease-Classification/src/clasifier_cnn/components/training.py", line 70, in train
    self.model.fit(
  File "/Users/rohithkamath/Documents/endtoendproject/chicken3/lib/python3.10/site-packages/keras/src/utils/traceback_utils.py", line 122, in error_handler
    raise e.with_traceback(filtered_tb) from None
  File "/Users/rohithkamath/Documents/endtoendproject/chicken3/lib/python3.10/site-packages/keras/src/optimizers/base_optimizer.py", line 329, in _check_variables_are_known
    raise ValueError(
ValueError: Unknown variable: <Variable path=dense/kernel, shape=(25088, 2), dtype=float32, value=[[ 0.011897   -0.00889055]
 [-0.00879171  0.01049202]
 [ 0.01110586  0.00455226]
 ...
 [-0.00044798  0.01047834]
 [-0.00620989  0.0112009 ]
 [ 0.00195403  0.00393406]]>. This optimizer can only be called for the variables it was originally built with. When working with a new set of variables, you should recreate a new optimizer instance.
[2025-05-09 17:59:03,599: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-09 17:59:03,602: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 17:59:03,604: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 17:59:03,605: INFO: common_func: created directory at: artifacts]
[2025-05-09 17:59:03,605: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-09 17:59:03,605: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-09 17:59:03,705: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-09 17:59:03,705: INFO: test_run: Trying to copy data set: This is the Second stage: Model Creation stay tuned and relax]
[2025-05-09 17:59:03,707: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 17:59:03,707: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 17:59:03,707: INFO: common_func: created directory at: artifacts]
[2025-05-09 17:59:03,707: INFO: common_func: created directory at: artifacts/prepare_base_model]
[2025-05-09 17:59:03,975: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 17:59:04,065: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 17:59:04,139: INFO: test_run: succefuly unziped the file and you are done with Second stage: Model Creation]
[2025-05-09 17:59:04,140: INFO: test_run: Trying to copy data set: This is the Third stage: Model Training stay tuned and relax]
[2025-05-09 17:59:04,142: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 17:59:04,142: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 17:59:04,143: INFO: common_func: created directory at: artifacts]
[2025-05-09 17:59:04,145: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-09 17:59:04,146: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-09 17:59:04,147: INFO: common_func: created directory at: artifacts/training]
[2025-05-09 17:59:04,356: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-09 17:59:44,102: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 17:59:44,210: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 17:59:44,291: INFO: test_run: succefuly unziped the file and you are done with Third stage: Model Training]
[2025-05-09 18:01:01,855: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-09 18:01:01,858: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 18:01:01,860: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 18:01:01,861: INFO: common_func: created directory at: artifacts]
[2025-05-09 18:01:01,861: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-09 18:01:01,861: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-09 18:01:01,960: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-09 18:01:01,960: INFO: test_run: Trying to copy data set: This is the Second stage: Model Creation stay tuned and relax]
[2025-05-09 18:01:01,961: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 18:01:01,962: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 18:01:01,962: INFO: common_func: created directory at: artifacts]
[2025-05-09 18:01:01,962: INFO: common_func: created directory at: artifacts/prepare_base_model]
[2025-05-09 18:01:02,226: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 18:01:02,327: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 18:01:02,399: INFO: test_run: succefuly unziped the file and you are done with Second stage: Model Creation]
[2025-05-09 18:01:02,399: INFO: test_run: Trying to copy data set: This is the Third stage: Model Training stay tuned and relax]
[2025-05-09 18:01:02,401: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 18:01:02,403: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 18:01:02,404: INFO: common_func: created directory at: artifacts]
[2025-05-09 18:01:02,404: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-09 18:01:02,404: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-09 18:01:02,404: INFO: common_func: created directory at: artifacts/training]
[2025-05-09 18:01:02,568: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-09 18:01:48,589: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 18:01:48,715: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 18:01:48,835: INFO: test_run: succefuly unziped the file and you are done with Third stage: Model Training]
[2025-05-09 18:07:09,648: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-09 18:07:09,652: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 18:07:09,654: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 18:07:09,654: INFO: common_func: created directory at: artifacts]
[2025-05-09 18:07:09,654: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-09 18:12:13,063: INFO: data_ingestion: succesfully downloaded the zip artifacts/data_ingestion/data.zip with following info 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 2333:1196:98F4:BEA0:681E2959
Accept-Ranges: bytes
Date: Fri, 09 May 2025 16:12:10 GMT
Via: 1.1 varnish
X-Served-By: cache-fra-etou8220108-FRA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1746807130.290741,VS0,VE545
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: c23b9d289768fb597cbc2dcdc571daebc3104309
Expires: Fri, 09 May 2025 16:17:10 GMT
Source-Age: 0

]
[2025-05-09 18:12:13,161: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-09 18:12:13,161: INFO: test_run: Trying to copy data set: This is the Second stage: Model Creation stay tuned and relax]
[2025-05-09 18:12:13,164: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 18:12:13,167: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 18:12:13,169: INFO: common_func: created directory at: artifacts]
[2025-05-09 18:12:13,169: INFO: common_func: created directory at: artifacts/prepare_base_model]
[2025-05-09 18:12:13,692: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 18:12:13,807: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 18:12:13,881: INFO: test_run: succefuly unziped the file and you are done with Second stage: Model Creation]
[2025-05-09 18:12:13,881: INFO: test_run: Trying to train the model: This is the Third stage: Model Training stay tuned and relax]
[2025-05-09 18:12:13,883: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-09 18:12:13,883: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-09 18:12:13,883: INFO: common_func: created directory at: artifacts]
[2025-05-09 18:12:13,884: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-09 18:12:13,884: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-09 18:12:13,884: INFO: common_func: created directory at: artifacts/training]
[2025-05-09 18:12:13,964: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-09 18:12:57,444: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 18:12:57,533: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-09 18:12:57,601: INFO: test_run: succefuly finished training Third stage: Model Training]
[2025-05-10 11:54:39,704: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-10 11:54:39,706: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 11:54:39,709: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 11:54:39,710: INFO: common_func: created directory at: artifacts]
[2025-05-10 11:54:39,710: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 11:54:39,711: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-10 11:54:39,873: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-10 11:54:39,873: INFO: test_run: Trying to copy data set: This is the Second stage: Model Creation stay tuned and relax]
[2025-05-10 11:54:39,874: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 11:54:39,875: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 11:54:39,875: INFO: common_func: created directory at: artifacts]
[2025-05-10 11:54:39,875: INFO: common_func: created directory at: artifacts/prepare_base_model]
[2025-05-10 11:54:40,192: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-10 11:54:40,468: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-10 11:54:40,870: INFO: test_run: succefuly unziped the file and you are done with Second stage: Model Creation]
[2025-05-10 11:54:40,870: INFO: test_run: Trying to train the model: This is the Third stage: Model Training stay tuned and relax]
[2025-05-10 11:54:40,872: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 11:54:40,874: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 11:54:40,874: INFO: common_func: created directory at: artifacts]
[2025-05-10 11:54:40,874: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-10 11:54:40,874: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-10 11:54:40,875: INFO: common_func: created directory at: artifacts/training]
[2025-05-10 11:54:41,056: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-10 11:55:33,883: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-10 11:55:34,155: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-10 11:55:34,316: INFO: test_run: succefuly finished training Third stage: Model Training]
[2025-05-10 12:02:41,900: INFO: test_run: Trying to copy data set: This is the First stage: data ingestion stay tuned and relax]
[2025-05-10 12:02:41,903: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 12:02:41,906: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 12:02:41,907: INFO: common_func: created directory at: artifacts]
[2025-05-10 12:02:41,907: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 12:02:41,907: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-10 12:02:42,030: INFO: test_run: succefuly unziped the file and you are done with First stage: data ingestion]
[2025-05-10 12:02:42,030: INFO: test_run: Trying to copy data set: This is the Second stage: Model Creation stay tuned and relax]
[2025-05-10 12:02:42,031: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 12:02:42,032: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 12:02:42,032: INFO: common_func: created directory at: artifacts]
[2025-05-10 12:02:42,032: INFO: common_func: created directory at: artifacts/prepare_base_model]
[2025-05-10 12:02:42,345: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-10 12:02:42,543: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-10 12:02:42,729: INFO: test_run: succefuly unziped the file and you are done with Second stage: Model Creation]
[2025-05-10 12:02:42,729: INFO: test_run: Trying to train the model: This is the Third stage: Model Training stay tuned and relax]
[2025-05-10 12:02:42,730: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 12:02:42,730: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 12:02:42,731: INFO: common_func: created directory at: artifacts]
[2025-05-10 12:02:42,731: INFO: common_func: created directory at: artifacts/prepare_callbacks/checkpoint_dir]
[2025-05-10 12:02:42,731: INFO: common_func: created directory at: artifacts/prepare_callbacks/tensorflow_log_dir]
[2025-05-10 12:02:42,731: INFO: common_func: created directory at: artifacts/training]
[2025-05-10 12:02:42,853: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-10 12:03:22,183: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-10 12:03:22,359: WARNING: saving_api: You are saving your model as an HDF5 file via `model.save()` or `keras.saving.save_model(model)`. This file format is considered legacy. We recommend using instead the native Keras format, e.g. `model.save('my_model.keras')` or `keras.saving.save_model(model, 'my_model.keras')`. ]
[2025-05-10 12:03:22,546: INFO: test_run: succefuly finished training Third stage: Model Training]
[2025-05-10 12:03:22,546: INFO: test_run: Trying to train the model: This is the Fourth stage: Model evaluation stay tuned and relax]
[2025-05-10 12:03:22,558: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 12:03:22,559: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 12:03:22,559: INFO: common_func: created directory at: artifacts]
[2025-05-10 12:03:22,665: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-05-10 12:03:36,339: INFO: common_func: json file saved at the location: scores.json]
[2025-05-10 12:03:36,340: INFO: test_run: succefuly finished training Fourth stage: Model evaluation]
[2025-05-10 12:13:05,783: INFO: stage01_data_ingestion: >>>>>> stage First Stage started <<<<<<]
[2025-05-10 12:13:05,786: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 12:13:05,787: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 12:13:05,788: INFO: common_func: created directory at: artifacts]
[2025-05-10 12:13:05,788: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 12:18:08,058: INFO: data_ingestion: succesfully downloaded the zip artifacts/data_ingestion/data.zip with following info 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 0A40:1619E2:766A19:8A95B9:681F2090
Accept-Ranges: bytes
Date: Sat, 10 May 2025 10:18:06 GMT
Via: 1.1 varnish
X-Served-By: cache-fra-etou8220042-FRA
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1746872286.324718,VS0,VE204
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 71f682350ed1601547e038354f42d034e6efad66
Expires: Sat, 10 May 2025 10:23:06 GMT
Source-Age: 0

]
[2025-05-10 12:18:08,147: INFO: stage01_data_ingestion: >>>>>> stage First Stage completed <<<<<<

x==========x]
[2025-05-10 12:20:06,631: INFO: stage01_data_ingestion: >>>>>> stage First Stage started <<<<<<]
[2025-05-10 12:20:06,634: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 12:20:06,638: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 12:20:06,639: INFO: common_func: created directory at: artifacts]
[2025-05-10 12:20:06,640: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 12:20:06,640: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-10 12:20:06,713: INFO: stage01_data_ingestion: >>>>>> stage First Stage completed <<<<<<

x==========x]
[2025-05-10 12:20:57,735: INFO: stage01_data_ingestion: >>>>>> stage First Stage started <<<<<<]
[2025-05-10 12:20:57,737: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 12:20:57,739: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 12:20:57,740: INFO: common_func: created directory at: artifacts]
[2025-05-10 12:20:57,740: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 12:20:57,740: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-10 12:20:57,815: INFO: stage01_data_ingestion: >>>>>> stage First Stage completed <<<<<<

x==========x]
[2025-05-10 12:21:24,331: INFO: stage01_data_ingestion: >>>>>> stage First Stage started <<<<<<]
[2025-05-10 12:21:24,333: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 12:21:24,336: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 12:21:24,337: INFO: common_func: created directory at: artifacts]
[2025-05-10 12:21:24,337: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 12:26:36,618: INFO: data_ingestion: succesfully downloaded the zip artifacts/data_ingestion/data.zip with following info 
Connection: close
Content-Length: 11616915
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "adf745abc03891fe493c3be264ec012691fe3fa21d861f35a27edbe6d86a76b1"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 0A40:1619E2:766A19:8A95B9:681F2090
Accept-Ranges: bytes
Date: Sat, 10 May 2025 10:26:25 GMT
Via: 1.1 varnish
X-Served-By: cache-fra-eddf8230127-FRA
X-Cache: HIT
X-Cache-Hits: 0
X-Timer: S1746872785.105377,VS0,VE153
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 0f6f14c49cf9270004727c8be43aab4990645a31
Expires: Sat, 10 May 2025 10:31:25 GMT
Source-Age: 0

]
[2025-05-10 12:26:36,744: INFO: stage01_data_ingestion: >>>>>> stage First Stage completed <<<<<<

x==========x]
[2025-05-10 13:30:59,857: INFO: stage01_data_ingestion: >>>>>> stage First Stage started <<<<<<]
[2025-05-10 13:30:59,860: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 13:30:59,863: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 13:30:59,864: INFO: common_func: created directory at: artifacts]
[2025-05-10 13:30:59,864: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 13:30:59,864: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-10 13:30:59,970: INFO: stage01_data_ingestion: >>>>>> stage First Stage completed <<<<<<

x==========x]
[2025-05-10 13:31:39,752: INFO: stage01_data_ingestion: >>>>>> stage First Stage started <<<<<<]
[2025-05-10 13:31:39,757: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 13:31:39,760: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 13:31:39,762: INFO: common_func: created directory at: artifacts]
[2025-05-10 13:31:39,763: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 13:31:39,763: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-10 13:31:40,011: INFO: stage01_data_ingestion: >>>>>> stage First Stage completed <<<<<<

x==========x]
[2025-05-10 13:32:11,213: INFO: stage01_data_ingestion: >>>>>> stage First Stage started <<<<<<]
[2025-05-10 13:32:11,215: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 13:32:11,217: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 13:32:11,218: INFO: common_func: created directory at: artifacts]
[2025-05-10 13:32:11,218: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 13:32:11,218: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-10 13:32:11,301: INFO: stage01_data_ingestion: >>>>>> stage First Stage completed <<<<<<

x==========x]
[2025-05-10 13:32:29,114: INFO: stage01_data_ingestion: >>>>>> stage First Stage started <<<<<<]
[2025-05-10 13:32:29,116: INFO: common_func: Yaml file: config/config.yaml has been loaded succesfully]
[2025-05-10 13:32:29,117: INFO: common_func: Yaml file: params.yaml has been loaded succesfully]
[2025-05-10 13:32:29,118: INFO: common_func: created directory at: artifacts]
[2025-05-10 13:32:29,118: INFO: common_func: created directory at: artifacts/data_ingestion]
[2025-05-10 13:32:29,118: INFO: data_ingestion: The file already exists and the size is  11345 KB]
[2025-05-10 13:32:29,211: INFO: stage01_data_ingestion: >>>>>> stage First Stage completed <<<<<<

x==========x]
